Test/Test_Ethernet.c

```diff
@@ -0,0 +1,357 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+/* Includes ------------------------------------------------------------------*/
+#include "main.h"
+#include "cmsis_os.h"
+#include "CANBus.h"
+#include "FreeRTOS.h"
+#include "queue.h"
+#include <stdio.h>
+#include <string.h>
+#include "radio.h"
+
```

Comment:
Most of these things in the test file don't really matter, but next time try to make sure the code is a bit clean (don't include things that aren't needed).